### PR TITLE
Address feedback items for chapter 1

### PR DESCRIPTION
**Pull request by chrishtr**
* Exercise added to chapter 2 for about:blank
* Drop assert error messages
* Removed extra commentary about HTML syntax
* Get rid of 200 assert (I really like this change!)
* Use casefold throughout all chapters
* Don't return headers from `request` until chapter 10

Also:
* Changed numbered footnotes into named ones, and put them inline

### CODE REVIEW

File: book/graphics.md [hunk 1]
```diff
@@ -591,6 +591,13 @@ window's new width and height can be found in the `width` and `height`
 fields on the event object. Remember that when the window is resized,
 the line breaks must change, so you will need to call `layout` again.
 
+*about:blank:* Currently, a malformed URL causes the browser to crash.
```
Comment (pavpanchekha) [inline book/graphics.md:594]: Any reason this goes in Chapter 2 instead of 1? Just because we need more in Chapter 2?

File: book/security.md [hunk 2]
```diff
@@ -792,9 +792,9 @@ def request(self, payload=None):
         params = {}
         if ";" in cookie:
             cookie, rest = cookie.split(";", 1)
-            for param_pair in rest.split(";"):
+        for param_pair in rest.split(";"):
```
Comment (pavpanchekha) [inline book/security.md:795]: Confused by this change. Either we need to initialize `rest` as the empty string, or the `for` loop needs to be inside the `if` statement. But maybe this code block was already misshapen? Or there's a spaces-vs-tabs issue?

**Review by pavpanchekha — APPROVED** [review]

File: book/graphics.md (hunk 1 above)
Comment (chrishtr) [reply book/graphics.md:594]: Yes: chapter 1 doesn't have a browser, it's just a commandline tool. It makes more sense to me to justify this when there is a browser UI to think about.

File: book/security.md (hunk 2 above)
Comment (chrishtr) [reply book/security.md:795]: This change was not on purpose, and seems to be a whitespace issue. Fixed.

**Pull request merged**